janusgraph-core/src/main/java/org/janusgraph/diskstorage/indexing/IndexMutation.java

```diff
@@ -33,19 +34,30 @@
 
 public class IndexMutation extends Mutation<IndexEntry,IndexEntry> {
 
+    private final KeyInformation.StoreRetriever storeRetriever;
     private final boolean isNew;
     private boolean isDeleted;
 
-    public IndexMutation(List<IndexEntry> additions, List<IndexEntry> deletions, boolean isNew, boolean isDeleted) {
+    private final Function<IndexEntry, Object> entryConversionFunction =
```

Comment:
Where is the apply method within this function? Consolidate still expects it to be present. If this doesn't cause a test failure we have a coverage issue we'll need to look into. 

Here's consolidate for reference:
https://github.com/JanusGraph/janusgraph/blob/1e64abc197f35292f0ba55eaa72a492208af3ad3/janusgraph-core/src/main/java/org/janusgraph/diskstorage/Mutation.java#L150-L153

Reply:
It's a syntax sugar of java 8 which allows to create anonymous single-method-class using lambda expression.

Reply:
> Where is the apply method within this function?

It is represented as lambda function. 

Looks like those are flaky tests. I've restarted them

Reply:
@chupman Tests passed

Reply:
Sorry for the delay, but I had to do some more testing and reading before I was comfortable approving.